openpype/settings/entities/schemas/projects_schema/schema_project_maya.json

```
@@ -153,6 +153,23 @@
         },
         {
             "type": "dict",
+            "collapsible": true,
+            "key": "update_instances",
+            "label": "Update animation instances on Reset Frame Range",
```

```suggestion
            "label": "Update publishable instances on Reset Frame Range",
```

Animation very much suggests that it only works for that family, but actually it updates all publishable instances